symbols-mui/README.md

```diff
@@ -1,12 +1,12 @@
-[![](https://img.shields.io/circleci/project/github/brightlayer-ui/symbols/master.svg?style=flat)](https://circleci.com/gh/brightlayer-ui/symbols/tree/master)
+[![](https://img.shields.io/circleci/project/github/etn-ccis/blui-symbols/master.svg?style=flat)](https://circleci.com/gh/etn-ccis/blui-symbols/tree/master)
 ![npm](https://img.shields.io/npm/v/@brightlayer-ui/symbols-mui?label=%40brightlayer-ui/symbols-mui)
 
 # Brightlayer UI Symbols for Material-UI
-This library contains componentized svg symbols from [@brightlayer-ui/symbols](https://github.com/brightlayer-ui/symbols) for use in React applications using Material UI. The symbols are made available in the same way that Material UI exposes the [Material Icons](https://material-ui.com/style/icons/#svg-material-icons). 
+This library contains componentized svg symbols from [@brightlayer-ui/symbols](https://github.com/etn-ccis/blui-symbols) for use in React applications using Material UI. The symbols are made available in the same way that MUI exposes the [Material Icons](https://mui.com/material-ui/icons/#svg-material-icons). 
 
 ## Installation
-To install the Brightlayer UI mui symbols from NPM as a dependency for your project, you can run one of the following commands in your project root:
-```
+To install the Brightlayer UI's MUI symbols from NPM as a dependency for your project, you can run one of the following commands in your project root:
```

Comment:
If you're adding the apostrophe here, it reads better without the "the" before Brightlayer
```suggestion
To install Brightlayer UI's MUI symbols from NPM as a dependency for your project, you can run one of the following commands in your project root:
```